Detectors/GlobalTracking/src/MatchTOF.cxx

```diff
@@ -1162,17 +1168,23 @@ void MatchTOF::doMatchingForTPC(int sec)
     minTrkTime = int(minTrkTime / BCgranularity) * BCgranularity;                                        // align min to a BC
     double maxTrkTime = (trackWork.second.getTimeStamp() + mExtraTPCFwdTime[cacheTrk[itrk]]) * 1.E6;     // maximum time in ps
 
-    /*
+    //   printf("trk time %f - %f (max shift +/- %f cm)\n",minTrkTime,maxTrkTime,trackWork.second.getTimeStampError()*vdrift );
+
+    std::array<float, 3> globalPos;
+    trackWork.first.getXYZGlo(globalPos);
+
```

Comment:
why is this needed? I see also in many places you do getXYZGlo (this is relatively expensive methods) just to use the result in LOG(DEBUG), i.s. almost never used. Since using LOG(DEBUG) is anyway possible only if the code was compiled with debug mode, perhaps also the getXYZGlo should be called in this case only?

Reply:
I guess it is a residual of my debug session (when I added a lot of prints and infos). Let me check

Reply:
Removed